### DISCUSSION TITLE
webMAN MOD 1.47.36

### DISCUSSION THREAD

**Discussion by aldostools** [aldostools/webMAN-MOD#536] [General]
🪐 Sign In to PSN now is blocked until CFW syscalls are disabled using R2+🔺  (Mod by @**DeViL303**)
🚀 Cumulative update (includes over 83 commits to the suite of tools after 1.47.35)

<hr>
💡 <b>If you like webMAN MOD, then feel free to contribute with a *small donation</font>*, I'd really appreciate it!<BR>
&nbsp;  &nbsp; &nbsp; <a href="https://www.paypal.me/aldostools/10">https://www.paypal.me/aldostools</a> (Funds are mainly to cover the high server costs)</b><br>

<hr /><em>This discussion was created from the release <a href='https://github.com/aldostools/webMAN-MOD/releases/tag/1.47.36'>webMAN MOD 1.47.36</a>.</em>

**Comment by GravuTrad** [comment]
Hello aldo. the more important isn't precised: 4.88 support or not? thanks

**Reply by CheezyFriez12** [reply]
Hey aldo, is there some way to modify the new psn blocker to spoof psid and idps to have that added layer of security against getting banned? Also, is there some way I can modify webman to let me use the web portal on another house's network? I have been traveling between home and my aunt's to help out and its pretty annoying.

**Reply by Masamune3210** [reply]
Pretty sure you can already spoof idps in the webman config, am I wrong?

**Reply by CheezyFriez12** [reply]
Im not sure, but it would be simpler to have it built in to the function.

**Reply by aldostools** [reply]
You can spoof idps and psid in http://0/setup.ps3

**Reply by CheezyFriez12** [reply]
Ok, what about using the web portal on another router

**Reply by aldostools** [reply]
webMAN runs a NET SERVER on the PS3 that can serve local content to remote consoles.
It only supports ISO in PS3ISO, BDISO, DVDISO or PSXISO.

Or you can setup a ps3netsrv on a PC.

**Comment by aldostools** [comment]
webMAN MOD supports current and future firmwares unless Sony makes big changes to LV1/LV2, which haven't happened since 4.75.

So, yes it supports 4.88

**Comment by GravuTrad** [comment]
thank you aldo.

**Comment by BRIAN-JESHUA-ZUKOSTORE** [comment]
hi aldostools, webman has some function like this? ..

If "/dev_hdd0/tmp/wallpaper/wallpaper.png" exists, it maps to "/ dev_hdd0 / home / CURRENT USER / wallpaper /"
replacing wallpapers through webman.
If it doesn't exist, I would like to propose it for future versions, or some way to pack wallpapers in pkg.
greetings and thank you very much

**Reply by aldostools** [reply]
webMAN MOD already has that function since the version 1.47.34

You can put up to 254 images in /dev_hdd0/tmp/wallpaper and name them 1.png, 2.png, 3.png etc,
The selected file will be copied to /dev_hdd0/home/000000xx/theme/wallpaper.png every time the XMB is loaded

Use the web command http://0/wallpaper.ps3?random to select a random wallpaper
or use http://0/wallpaper.ps3?(ID) to always load one specific image

**Reply by BRIAN-JESHUA-ZUKOSTORE** [reply]
Oh thank you aldo 😃

**Comment by ghost** [comment]
> 
> 
> 🪐 Sign In to PSN now is blocked until CFW syscalls are disabled using R2+🔺 (Mod by @**DeViL303**)

Hi aldostools, i would like to go online with enabled syscalls how can i bypass the R2+🔺 procedure?

**Reply by aldostools** [reply]
> > 🪐 Sign In to PSN now is blocked until CFW syscalls are disabled using R2+🔺 (Mod by @**DeViL303**)
> 
> Hi aldostools, i would like to go online with enabled syscalls how can i bypass the R2+🔺 procedure?

Delete the rco in /dev_hdd0/tmp/wm_res 
Then reboot

**Reply by ghost** [reply]
Thank you very much, it worked. It would be interesting to be able to activate it from /setup.ps3 with a checkbox.
Two more questions: 
- is it possible to manage the wifi proxy settings directly from webman mod's /setup.ps3 without having to reconfigure the network settings every time? 
- is it possible to add a pad shortcut or a display in xmb that show the current mac address?
Thanks in advance!

**Reply by aldostools** [reply]
> * is it possible to manage the wifi proxy settings directly from webman mod's /setup.ps3 without having to reconfigure the network settings every time?
> * is it possible to add a pad shortcut or a display in xmb that show the current mac address?
>   Thanks in advance!

No. You must configure the proxy through Network Settings. 
I recommend to configure the proxy and the PS3 with static IP addresses, if you haven't already.

The MAC address is shown in /cpursx.ps3 
The combo L2+R2+L1+O opens the browser in this page.

**Comment by Frikortreat** [comment]
is PrepISO already included in the Webman mod installation or we need to install it by ourselves?

**Comment by aldostools** [comment]
It is included in the installation, but you have to install it by yourself.

You can install the webman addons using http://0/install.ps3$

Or you can download prepISO from the release page or from webMAN MOD's page in brewology.com

**Comment by jhonnyn5** [comment]
Hello man please help me
I have a problem, you will see for x reasons I had to rebuild the "file system from recovery" and now I have the error that every time I start a ps2 game in pkg format when exiting again to xmb webman it slows me down the  ps3, it is a very considerable lag and it does not go away until I restart the console, I already tried to reinstall, reinstall the webman installer and even format the ps3 to see if it worked that way but not that bug is still there.  I am currently on CFW 4.88 evilnat in the official version not the beta. And latest webman PD: I know it is webman because after uninstalling it I have no problem playing my games

**Comment by aldostools** [comment]
Try updating to the latest 4.88.2 or downgrade to any Rebug version and test again.

There is a known memory leak in Evilnat 4.88 (non beta) loading the Cobra payload that could be causing random issues.

**Reply by jhonnyn5** [reply]
Sorry for the very late answer, but the bug is still there I tested with evilnat 4.88.2 beta and final version.
There is a lot of lag after playing a ps2 pkg and you have to restart the console to fix it

**Reply by aldostools** [reply]
If the lag is playing a PS2 game, it could be a compatibility issue with the game.

Take in count that PS2 games are emulated on almost all PS3 models except some early phat models that have a PS2 hardware inside. That means that not all games will play perfect. Some have glitches or lags, and others will not even boot.

Many PS2 games can be fixed with a CONFIG file. However some CONFIGs only make the game playable with lags or glitches.
You should try a different game or try to find a fix in https://ps3.aldostools.org/ps2config.html

Or try to find help in this forum:
https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/

**Reply by jhonnyn5** [reply]
I do not mean that man, I know that ps2 games do not run well at all, I mean that after exiting the game to the XMB the console has a lot of LAG and the only thing that fixes it is to restart the console and it is because of the webman because when not installed I have no error thanks I will try it anyway better if you check this error on evilnat 4.88.2

**Reply by aldostools** [reply]
Please verify that you have checked the option "Disable content scan on startup" on /setup.ps3 
If it is not checked, mark it and save the settings. That option can be the cause of lags when you return to XMB from a PS2 game.

**Reply by jhonnyn5** [reply]
I did it man, but it didn't work it's pretty weird, hopefully you could find out if it happens to other users

**Reply by jhonnyn5** [reply]
PD: I don't have any settings changed in webman I use the default one.  I only have the fan set to manual at 35%

**Reply by aldostools** [reply]
The setting "Disable content scan on startup" is unchecked by default to let new users find the games automatically.
However it is recommended to disable the content scan on startup because it may cause lag issues due the high memory consumption.

I would say that try a couple of times more just to be sure. Also double check that the option is really checked.

At the moment I cannot verify or fix anything due my PS3 died 5 days ago.

**Reply by jhonnyn5** [reply]
I'm sorry to hear that man, okay I'll make a couple more attempts, I'll write tomorrow or in a couple of days to see if the error is solvable

**Reply by jhonnyn5** [reply]
Hi, man, thanks for all the help, I was able to solve the error.  I don't know why but this error only happens when you use generic DS3s if you use original controls it works fine.  I don't know why this happens, the truth is very rare, but I clarify that the control is not lagging, the system is lagging.  Conclusion: better to use original controls and not have problems thanks again I have no problem with the default webman configuration.
Thanks for your time.

**Reply by aldostools** [reply]
I'm glad that you could fix the issue. 

These cheap DS3s usually have loose buttons or springs too soft and may perform involuntary combos.

I'm not saying that it's your case, but it is a potential cause of your lagging.

**Comment by nathannichols** [comment]
Wouldn't be able to say if it's an issue more related to WebMan over Evilnat's 4.88.2 CFW, but the PSP launchers seems to both corrupt after a successful launch or two of either one without fail. Then become impossible to delete via FTP or XMB, though trying a few times through the new file browser in Evilnat's CFW and rebuilding the database twice usually gets rid of the corrupted data. Happens all over again after reinstalling them from that point. Though obviously compatibility was always spotty I never had the same corruption issues on the previous version of WebMan while running Rebug 4.84.2 with CECHE01 model console.

No change after trying to mount a PSP game while showing as corrupted, process fails outright. And am unable to reinstall from .pkg files. It recognizes there's the same version currently installed but then errors out while saying "The USB device cannot be accessed".

**Comment by aldostools** [comment]
The PSP Launchers were updated recently due I published by mistake a corrupted version. 
Also the PSP Launchers now are blocked until you mount a PSPISO.

If you need to reinstall the PSP Launchers, you should mount a PSPISO then delete the PSP Launchers or install the new version.

If you try to reinstall the PSP Launchers while they are still blocked, you will receive the messae "The USB device cannot be accessed".

**Reply by nathannichols** [reply]
While blocked are they supposed to show up as corrupted data? That all made sense to me understanding the way they're supposed to work from what I was reading. BUT, from what I can tell, once they're corrupted I can't mount a PSP ISO at all. OR sometimes just the non remastered launcher shows up correctly again. I'll have to experiment and fresh install them. Maybe I just got over zealous with how and when I was considering something broken and attempting an uninstall. Really appreciate the response though! And obviously, great work on everything.

UPDATE: Yeah, I must've just done something outside of the correct order of operations. When mounting a PSP ISO even if the emulators still say corrupted because of me doing something incorrectly I can at least delete them and reinstall from the packages as expected. So I think that's all working exactly as intended. Awesome!

**Reply by Zoukambada** [reply]
@aldostools , @nathannichols also can use the command http://0/remap.ps3 to unlock PSP Launchers and then you will be able uninstall them and reinstall the PKGs.

**Comment by Ordeit** [comment]
Where to push config file ps2. So that the ps2 game will automatically pick it up. And whether the names of the game and the config should somehow match, discard the faq or explain. My game config does not pick up

**Comment by aldostools** [comment]
For CFW:

/dev_hdd0/PS2ISO/Game Name [SLES_123.45].ISO
/dev_hdd0/PS2ISO/Game Name [SLES_123.45].ISO.CONFIG

For HEN or CFW:

/dev_hdd0/PS2ISO/Game Name [SLES_123.45].BIN.ENC
/dev_hdd0/PS2ISO/Game Name [SLES_123.45].BIN.ENC.CONFIG

You can download the CONFIG and configure it manually from:
https://ps3.aldostools.org/ps2config.html

If the game id is included in the file name like in the example, webMAN will use the CONFIG from the database.
For CFW the CONFIGs are included in webMAN installation.
For HEN the standalone package for PS2 CONFIG database must be installed.

**Reply by Zoukambada** [reply]
If the game is
Game Name [SLES12345].ISO
No underscores, just the plain SERIALNUMBER between brackets, will it work too? I know the covers will load, what about the PS2 Configs?

**Reply by aldostools** [reply]
I will work with any name (even without the game ID) if you provide the cover and CONFIG file from https://ps3.aldostools.org/ps2config.html

Example:
/dev_hdd0/PS2ISO/Game Name.ISO
/dev_hdd0/PS2ISO/Game Name.ISO.CONFIG
/dev_hdd0/PS2ISO/Game Name.JPG (or .PNG)

The game ID is used only to find the cover and CONFIG in their respective databases (when the cover or CONFIG are not provided).
The game ID can in these format: [SLES12345], [SLES-12345] or [SLES_123.45].

The game ID is also detected if it is include at the begining of the file name: 
SLES-12345-[Game Name].ISO

**Comment by auronpl315** [comment]
Please fix ISO Valkyrie Profile 2 Silmeria (4.66 GB) ID see photo. Crash game, when long play.

![Screenshot 2021-08-08 at 20 17 54](https://user-images.githubusercontent.com/72985338/128641708-7217aa31-311e-46e7-ab11-40f33ab24b84.png)

**Reply by aldostools** [reply]
Try adding one of CONFIG files created for that game
https://ps3.aldostools.org/ps2config.html?q=silmeria

Example:
/dev_hdd0/PS2ISO/VP2.ISO  <- Your ISO for Valkyrie Profile 2 Silmeria
/dev_hdd0/PS2ISO/VP2.ISO.CONFIG <- CONFIG file downloaded from the link above

If the bug persists, try reporting the issue in this forum:
https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/

**Reply by auronpl315** [reply]
> Try adding one of CONFIG files created for that game
> https://ps3.aldostools.org/ps2config.html?q=silmeria
> 
> Example:
> /dev_hdd0/PS2ISO/VP2.ISO <- Your ISO for Valkyrie Profile 2 Silmeria
> /dev_hdd0/PS2ISO/VP2.ISO.CONFIG <- CONFIG file downloaded from the link above
> 
> If the bug persists, try reporting the issue in this forum:
> https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/

Okay will check.

**Comment by auronpl315** [comment]
Doesnt work config, game crash off screen. Console still green light and pad red light P1.

**Comment by aldostools** [comment]
If the bug persists, try reporting the issue in this forum:
https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/

**Comment by anonimous3243** [comment]
Hi aldostools, I just wanted to let you know that for some reason the new version is causing my Hen and my video game to crash and freeze, so I'm not able to use Webman, I'll be using old versions until you solve this problem. Anything or if you already have the solution for my problem, please tell me, I'm accepting help to solve this

**Comment by aldostools** [comment]
@anonimous3243  Thank you for your feedback. I think this issue could be related to the recent addition of backup act.dat
I've removed that change until I can test it. You can use the online updater via XMB to install that release.

Please report back if it solved your issue.

**Reply by Vitor240885** [reply]
I can't get into the webman settings. every time I go in it shows that a page with these hots was not found

**Comment by Vitor240885** [comment]
I can't get into the webman settings. every time I go in it shows that a page with these hots was not found

**Comment by aldostools** [comment]
Check if the plugin is running. Prsss select+start.
Maybe it was not loaded by boot_plugins.txt

**Comment by BRIAN-JESHUA** [comment]
hello aldo, sorry for bothering you but... is there a web command for the Getting QA Flag Menu?

**Reply by aldostools** [reply]
> hello aldo, sorry for bothering you but... is there a web command for the Getting QA Flag Menu?

There is not a specific web command for that, but if you set the focus on the column Settings > Network Settings
Then press r1+r2+l1+l2+l3+down at the same time you will get the QA flag menu,

Or you can create a shortcut on a PC, tablet or smartphone and execute this command if you already have the focus on Network Settings.
http://&lt;ip-address-of-ps3>/pad.ps3?r1+r2+l1+l2+l3+down+hold

**Reply by BRIAN-JESHUA** [reply]
Is there any way to install hfw update from hdd0? It is very useful for the user and I did not find much about it..

**Reply by aldostools** [reply]
> Is there any way to install hfw update from hdd0? It is very useful for the user and I did not find much about it..

That is a feature officially available when you enable QA flags on CFW and enable debug update in the QA debug menu.
The system looks in this folder: /dev_hdd0/updater/01

However on HFW, that feature is unavailable due HFW cannot enable QA flags (which also allows downgrade without hardware).

I haven't tried if it works or not, but in emer_init.self there are references to /dev_hdd0/vsh/PS3UPDAT.PUP and to /dev_hdd0/game/SCEEXE000/ARCDIR/PS3UPDAT.PUP

From XMB, I didn't find full paths in software_update_plugin.self, but there are references to /dev_hdd0 and /vsh along other paths that you could try placing the PUP inside these folders:
- /updater
- /push
- /PS3UPDAT.TMP
- /RESUME.TMP
- /pup_check.tmp
- /pup_check.dat
- /PS3UPDATE
- /PS3/UPDATE
- /PS3_UPDATE
- /vsh
- /SEARCH

I cannot guarentee that they will work, but you could try if you like to experiment. Although I do not recommend to mess with system files unless you have a hardware flasher.

**Reply by BRIAN-JESHUA** [reply]
Thanks for replying. It has been a great help. I'll see if I'm lucky..

**Reply by BRIAN-JESHUA** [reply]
I found a way.. which requires activate System Update Debug option from the QA flag menu..
My question is webman, could you bring a web command to activate System Update Debug?
That would be very useful for the common user.
by activating System Update Debug you will be able to search for .pup files in dev_hdd0/updater/01/ it would no longer depend on a USB

**Comment by aldostools** [comment]
This web command changes the setting, but requires to restart the system to let the XMB read the change in the setting.
```
http://0/xmb.ps3$xregistry(/setting/system/debugSystemUpdate)=1
```

Update before try it. While I was testing I found a bug in this web command. It is fixed in the latest commit.

**Reply by BRIAN-JESHUA** [reply]
thanks aldo.. it will be a great help for the community
